db/migrate/20230518170703_add_description_to_contributions.rb

```diff
@@ -0,0 +1,5 @@
+class AddDescriptionToContributions < ActiveRecord::Migration[7.0]
+  def change
+    add_column :contributions, :description, :text
```

Comment:
almost sure this was added in another PR, merged Friday, can you check?

Reply:
It was. Keeping only the older migration now